ci/scripts/release.sh

```diff
@@ -92,6 +94,10 @@ if [[ -n "${BUILDKITE_TAG}" ]]; then
 
   echo "--- Release build and upload risingwave connector node jar asset"
   gh release upload "${BUILDKITE_TAG}" risingwave-connector-"${BUILDKITE_TAG}".tar.gz
+
+  echo "--- Release upload risingwave-all-in-one asset"
+  tar -czvf risingwave-all-in-one-"${BUILDKITE_TAG}"-x86_64-unknown-linux.tar.gz risingwave libs
```

Comment:
Do we consider setting a default lib path relative to the risingwave binary in jvm initialization so that users don't need to explicitly set the env var when they want to extract the tar and run directly?

Reply:
Make senses. We can use a default lib path when the environment variable `CONNECTOR_LIBS_PATH` is not set.